xml/hwbp_registers.xml

```diff
@@ -220,6 +220,19 @@
         hold all valid virtual addresses but it need not be capable of holding
         other values.
 
+        If the A extension is supported, then trigger behavior is as follows
+        for the load and store bits:
+        \begin{tightlist}
+        \item LR instructions are loads
```

Comment:
LR -> {\tt lr}

Reply:
I was using all-caps since that's what the unprivileged and privileged specs do when referring to instructions.  They use lower-case only in code snippets.  Except in appendix B of the unprivileged spec.

But I'll change it to be consistent with the rest of this document.  Maybe someday RVI will have a techpubs department that gets everything to have a similar look and feel across documents and, as with appendix B, within the same document.